base/ca/shared/conf/CS.cfg

```diff
@@ -23,6 +23,7 @@ preop.system.name=CA
 preop.system.fullname=Certificate Authority
 proxy.securePort=[PKI_PROXY_SECURE_PORT]
 proxy.unsecurePort=[PKI_PROXY_UNSECURE_PORT]
+ca.use_oaep_rsa_key_wrap=false
```

Comment:
How about "ca.keyWrap.useOAEP=false" ? Does KRA have similar config too?  Or maybe just "keyWrap.useOAEP=false"?

Reply:
It should work, but I have some comments about this:

* If this parameter is meant to be used in all subsystems we might want to drop the `ca.` prefix.
* If we add a new param in CS.cfg we should add an upgrade script to update existing instances.
* If the param has a default value, it doesn't have to be added into CS.cfg, and we don't need to add an upgrade script.
* Instead of adding a specific boolean parameter for OAEP, should we add a param to specify the algorithm name so it can be used for other key wrap algorithms in the future?